modules/components/pages/inputs/redpanda_migrator.adoc

```diff
@@ -705,6 +708,32 @@ The label of the `redpanda_migrator` output in which the currently selected topi
 
 *Type*: `string`
 
-*Default*: `"redpanda_migrator_output"`
+*Default*: `redpanda_migrator_output`
+
+=== `replication_factor_override`
```

Comment:
I'd move `output_resource`, `replication_factor_override`, and `replication_factor` higher up the hierarchy so it's closer to `topics`.  These are all related to how the connector interacts with topics but it's in the same place as low-level batching settings.

Reply:
Updated as requested.